scripts/index.js

```diff
@@ -365,5 +390,7 @@ function stopSendingPeriodicMessage() {
         start: true,
         stop: false
       });
+
+      displayErrorMsgBox("Server error");
```

Comment:
`start` and `stop` should display their own error message as discussed [here](https://github.com/c-hive/ws-anywhere/issues/19#issuecomment-498015808). If I understand it correctly this is not the case right now. Could you provide a deployment where the server is broken and this can be tested?

Reply:
That's not part of the current PR - those icons will be implemented separately. Sorry if the description wasn't that precise.

What the PR actually does is display the `Server error` message in the case of BE errors and also fix the previous fatal error.

> Could you provide a deployment where the server is broken and this can be tested?

Yes.

Reply:
> display the Server error message in the case of BE errors

This should only be scoped to "global" errors. If there's an error regarding a user interaction it should be displayed in place.

So I think these 2 occurrences could be removed and then done in a separate PR, no?

Reply:
Alright, you might be right.

So, you mean the start/stop button related errors are specific ones, right? Thereby, they should be displayed inline - or in place, as you said above.

Reply:
However, what if the error occurs during the [**MongoDB connection**](https://github.com/c-hive/ws-anywhere/blob/5904d148ab23544632252aa34ad6beb57d9a9cc9/app.js#L15-L39)?

Reply:
Error message box demonstration is available here:
- **https://c-hive-ws-demo.herokuapp.com/**

Reply:
Discussed on the phone.